practice-one/src/components/Sort/Sort.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./Sort.css";
+import { arrowDown } from "@assets/images";
+
+const Sort = ({ sortName }) => {
```

Comment:
Sort it's mean a function. But on this component, it isnot have sort. So should update name

Reply:
I changed from sort to expand